pennylane/hf/tapering.py

```diff
@@ -464,3 +465,65 @@ def transform_hamiltonian(h, generators, paulix_ops, paulix_sector):
     c = qml.math.stack(c)
 
     return _simplify(qml.Hamiltonian(c, o))
+
+
+def optimal_sector(qubit_op, generators, active_electrons):
+    r"""Get the optimal sector which contains the ground state.
+
+    To obtain the optimal sector, we need to choose the right eigenvalue for the symmetries :math:`\vec{\tau}`. We can do
```

Comment:
It seems to me that the optimal eigenvalue sector is obtained by applying each `\tau` to the referee HF state. Assuming that `\tau`s are composed of only I and Pauli-Z, then finding the eigenvalue is trivial by just looking at the wires of the Pauli-Z ops and the occupation state of the qubits.

Reply:
Thanks for the confirmation on this! I have updated the docstring to make it more concise.